spec/requests/groups_controller_spec.rb

```diff
@@ -858,6 +858,54 @@ def expect_type_to_return_right_groups(type, expected_group_ids)
 
       context "user default notifications" do
         it "should update default notification preference for existing users" do
+          user1 = Fabricate(:user)
+          user2 = Fabricate(:user)
+          group.add(user1)
+          group.add(user2)
+
+          put "/groups/#{group.id}.json", params: {
+            group: {
+              default_notification_level: 2
+            }
+          }
+
+          expect(response.status).to eq(200)
+          group_users = group.group_users
+          expect(response.parsed_body["user_count"]).to eq(group_users.count)
```

Comment:
Since the description of this test is about updating the notification level, I think the assertions here need to assert that the desired notification level has been updated. I'm not quite sure what `user_count` represents but I do not think it is sufficient to test for a number of user that has been changed.